elasticdl/python/worker/worker.py

```diff
@@ -306,7 +306,14 @@ def get_task(self, task_type=None):
         if task_type is not None:
             req.task_type = task_type
 
-        return self._stub.get_task(req)
+        try:
+            res = self._stub.get_task(req)
```

Comment:
When will this case happen? IMO, if master stops, the worker should already stopped.

Reply:
The worker may get  `WAIT` task, and continue to  `get_task`.  
When in a master thread, it checks task status every 30 seconds. When it detects that there will be no more tasks, the master service will stop immediately. If by chance that a worker get a `WAIT` task, and before it calls the next `get_task`, the master service stops, the next `get_task` will get exception.

Reply:
Thanks for the detailed explain, how about adding this information in the comments of the code block?

Reply:
comments added.